### PR TITLE
Adding html to website

### PR DESCRIPTION
**Pull request by jem1520**
These are my html pages for the adopt a dog website.

### CODE REVIEW

**Comment by jem1520** [comment]
I've added my updated CSS in this pull request. Thanks!

**Comment by jem1520** [comment]
I've added my javascript to this pull request.

**Comment by jem1520** [comment]
I've added media queries to the home page.

File: styles/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+       }
+   }
+   @media screen and (min-width: 200px) {
+       img {
+         display: inline;
+          width: 100%;
+       }
+   }
+   @media screen and (max-width: 768px) {
+        .hidetext {
```
Comment (tylervanbrocklin) [inline styles/stylesheet.css]: You'll want to take a closer look here. Per the assignment, we're looking for the blog previews on the home page to disappear when on mobile. You're on the right track with using display. Feel free to reach out if you need some help on this!

File: styles/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+          display: none;
+       }
+   }
+
+   @media screen and (max-width: 768px) {
+        .box-content1 {
```
Comment (tylervanbrocklin) [inline styles/stylesheet.css]: On this one, we're looking for you to manually set the grid columns within the media queries, as opposed to using auto and setting a manual width.

**Comment by jem1520** [comment]
I've added my updated media queries to this pull request.

File: styles/stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,458 @@
...
+            object-fit: contain;
+        }
+    }
+    @media screen and (min-width: 200px) {
+        .blogimage2 {
+          height: 200px;
+          object-fit: contain;
+          padding-top: 30px;
+        }
+    }
```
Comment (tylervanbrocklin) [inline styles/stylesheet.css]: I've noticed you have lots of duplicate @media queries. There is no limit on how much CSS you put into one query.

File: styles/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,458 @@
...
+        .grid-container {
+            grid-template-columns: 1fr;
+        }
+    }
+    @media screen and (max-width: 576px) {
+        h1 {
+            font-size: 50%;
+        }
+    }
+    @media screen and (min-width: 576px) {
```
Comment (tylervanbrocklin) [inline styles/stylesheet.css:75]: You can combine max-width and min-width onto the same query. As an example:

`@media only screen and (min-width:576) and (max-width:768)`

**Review by tylervanbrocklin — COMMENTED** [review]
Looks good! Would just clean up some of the extra/redundant @media queries. Well done on this one.

**Comment by jem1520** [comment]
I've added my new header styling for this assignment.

File: styles/stylesheet.css (hunk 3 above)
Comment (jem1520) [reply styles/stylesheet.css]: Thanks! I tried to clean up my media queries some with assignment 7.

File: styles/stylesheet.css (hunk 4 above)
Comment (jem1520) [reply styles/stylesheet.css:75]: Thanks! I tired to do this, but I struggled with the syntax. How do you code a combine max-width and min-width query if the css within the two separate queries are different?

**Comment by jem1520** [comment]
Adding lesson 8 assignment to this pull request.

**Comment by jem1520** [comment]
Adding jquery to website for lesson 9 assignment.

**Comment by jem1520** [comment]
Updating blogpage javascript to make blog page responsive again.

File: javascript/checkoutjq.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+
+
+$('.submitbutton').click(function(e) {
+   {checkOut();}
```
Comment (tylervanbrocklin) [inline javascript/checkoutjq.js:4]: would include: `e.preventDefault()` to prevent the page from refreshing

File: javascript/checkoutjq.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  let email = $('#email').val();
+  let address = $('#address').val();
+  let city = $('#city').val();
+  let state = $('#state').val();
+  let zip = $('#zip').val();
+  let radio = $( "input:checked" ).val()
+  let location = $('#location').val();
+
+
+  console.log (name + "|" + email + "|" + address + "|" + city + "|" + state + "|" + zip + "|" + radio + "|" + location);
```
Comment (tylervanbrocklin) [inline javascript/checkoutjq.js:20]: would be good to include labels with your output here.

File: javascript/blogpage.js [hunk 7]
```diff
@@ -0,0 +1,149 @@
+
+const blogitems = [
+{
+  img: 'images/blog-1.jpg',
+  alt: 'image from behind a women hugging black dog while sitting and looking at a beautiful view of a cayon',
+  h3: 'Traveling With Your Dog',
+  p: 'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat. Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+}];
+const blogitems2 = [
```
Comment (tylervanbrocklin) [inline javascript/blogpage.js:9]: Also sent in email, but you should combine these three into one array of blogItems.

```
const blogitems = [
{
  img: 'images/blog-1.jpg',
  alt: 'image from behind a women hugging black dog while sitting and looking at a beautiful view of a cayon',
  h3: 'Traveling With Your Dog',
  p: 'text here'
},
{
  img: 'images/blog-1.jpg',
  alt: 'image from behind a women hugging black dog while sitting and looking at a beautiful view of a cayon',
  h3: 'Traveling With Your Dog',
  p: 'text here'
},
{
  img: 'images/blog-1.jpg',
  alt: 'image from behind a women hugging black dog while sitting and looking at a beautiful view of a cayon',
  h3: 'Traveling With Your Dog',
  p: 'text here'
}
];
```

**Review by tylervanbrocklin — COMMENTED** [review]
Looks great, see my comments for suggestions on how to improve here. Reach out if you have any questions.

**Comment by jem1520** [comment]
Adding lesson 10 assignment.

**Pull request closed without merging**